recipes/default.rb

```diff
@@ -105,4 +107,5 @@
   #{gpg_command} --batch --gen-key $TMP_REQUEST
   EOH
   not_if { ::File.exist?("#{node['aptly']['rootDir']}/.gnupg/trustdb.gpg") }
+  rm -f $TMP_REQUEST
```

Comment:
yeah, you just need to remove this line. It looks like you're trying to run back, but you're actually in a Ruby block!

Reply:
Makes sense, for somehow i've chosen the wrong line - so i corrected it. The change does work on my setup.